starport/pkg/cosmosgen/templates/vuex/store/index.ts.tpl

```diff
@@ -92,55 +103,59 @@ export default {
 				dispatch(subscription.action, subscription.payload)
 			})
 		},
-		{{ range .Module.HTTPQueries }}async {{ .FullName }}({ commit, rootGetters, getters }, { options: { subscribe = false , all = false}, params: {...key}, query=null }) {
+		{{ range .Module.HTTPQueries }}
+		{{ $FullName := .FullName }}
+		{{ $Name := .Name }}
+		{{ range $i,$rule := .Rules}} 
+
+		async {{ $FullName }}{{ if (gt $i 0) }}{{$i}}{{ end}}({ commit, rootGetters, getters }, { options: { subscribe, all} = { subscribe:false, all:false}, params: {...key}, query=null }) {
 			try {
+				const queryClient=await initQueryClient(rootGetters)
+				let value= (await queryClient.{{ camelCase $FullName }}({{ range $j,$a :=$rule.Params}}{{ if (gt $j 0)}}, {{ end}} key.{{$a}}{{end}} {{ if $rule.HasQuery }}{{ if $rule.Params }}, {{ end }}query{{ end }}{{ if $rule.HasBody}}{{ if $rule.HasQuery }},{{ end }}{...key}{{ end }} )).data
```

Comment:
Might be missing a comma.

```js
let value= (await queryClient.msgApiDepositWithinBatchApi( key.pool_id *HERE* {...key} )).data
```

```
gitpod /workspace/starport/docs/liquidity $ starport serve
Cosmos SDK's version is: Stargate v0.40.0 (or above)

🛠️  Building proto...
cannot build app:

        error while running command /tmp/nodetime: : exit status 1

js/generated/tendermint/liquidity/tendermint.liquidity/index.ts(255,76): error TS1005: ',' expected.
js/generated/tendermint/liquidity/tendermint.liquidity/index.ts(275,77): error TS1005: ',' expected.
js/generated/tendermint/liquidity/tendermint.liquidity/index.ts(295,62): error TS1005: ',' expected.
```

Reply:
dammit...didn't check the latest change with liquidity...it needs to add the comma if hasQuery OR if has Params...will fix in a sec